PayItForward/PayItForward.ConsoleClient.UnitTests/BasicLoggerInfo/PrintInfo_Should.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace PayItForward.ConsoleClient
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using Xunit;
+
+    public class PrintInfo_Should
```

Comment:
You can add one more test to check if the Print method from the console wrapper is called with the proper text